batch/batch/front_end/templates/billing_projects.html

```diff
@@ -39,7 +39,7 @@ <h1>Billing Projects</h1>
       <tr>
         <form action="{{ base_path }}/billing_projects/{{ billing_project['billing_project'] }}/users/add" method="POST">
           <input type="hidden" name="_csrf" value="{{ csrf_token }}">
-          <td><input type="text" name="user"></td>
+          <td><input type="text" name="user" autocomplete="off"></td>
```

Comment:
```suggestion
          <td><input type="text" name="user" spellcheck="false" autocorrect="off"></td>
```

see my comments on https://github.com/hail-is/hail/pull/13497; we actually want `spellcheck="false" autocorrect="off"` here, not `autocomplete="off"`

tested that out in safari with the new devserver from https://github.com/hail-is/hail/pull/13632 just now to confirm